apps/web/src/pages/Applications/ApplicationResumePage/ApplicationResumePage.tsx

```diff
@@ -51,15 +82,395 @@ export const getPageInfo: GetApplicationPageInfo = ({
+                    {formatExperienceCount(
+                      intl,
+                      experienceType as ExperienceType,
+                      experiencesByType[experienceType].length,
+                    )}
+                  </li>
+                );
+              })}
+            </ul>
+          </p>
+        </>
+      ) : (
+        <>
+          <p data-h2-margin="base(x1, 0)">
+            {intl.formatMessage({
+              defaultMessage:
+                "Creating your résumé is a little different on this platform. First and foremost, any work you do here will be saved to your profile so that you can reuse it on other applications down the road.",
+              id: "19hs6x",
+              description:
+                "Application step to begin working on résumé, paragraph one",
+            })}
+          </p>
+          <p data-h2-margin="base(x1, 0)">
+            {intl.formatMessage({
+              defaultMessage:
+                "Building your résumé consists of describing <strong>work experiences</strong>, <strong>education</strong>, <strong>community participation</strong>, <strong>personal learning</strong>, and <strong>awards</strong> you’ve earned. In a later step, you’ll use these experiences to highlight your skills. You can start adding experiences to your résumé using the “<strong>Add a new experience</strong>” button.",
+              id: "TEwoWv",
+              description:
+                "Application step to begin working on résumé, paragraph two",
+            })}
+          </p>
+        </>
+      )}
+
+      <FormProvider {...methods}>
+        <form onSubmit={methods.handleSubmit(handleSubmit)}>
+          <div
+            data-h2-display="base(flex)"
+            data-h2-flex-direction="base(column) p-tablet(row)"
+            data-h2-justify-content="base(space-between)"
+            data-h2-align-items="base(flex-start) p-tablet(center)"
+          >
+            <Select
+              id="sortExperiencesBy"
+              label={intl.formatMessage({
+                defaultMessage: "Sort your experiences by",
+                id: "XuYilo",
+                description: "Label for experience sort dropdown",
+              })}
+              name="sortExperiencesBy"
+              options={[
+                {
+                  value: "date_desc",
+                  label: intl.formatMessage({
+                    defaultMessage: "Date (recent first)",
+                    id: "narnyP",
+                    description: "Sort experiences by date descending",
+                  }),
+                },
+                {
+                  value: "type_asc",
+                  label: intl.formatMessage({
+                    defaultMessage: "Type",
+                    id: "m7tUAm",
+                    description: "Sort experiences by type ascending",
+                  }),
+                },
+              ]}
+              hideOptional
+              trackUnsaved={false}
+            />
```

Comment:
I don't think anything is actionable here but I'm a little confused as to how sorting by type works unless I read the code. As a user, I don't think this is useful, how is a user meant to know how this will function and in what scenario would it be used? 🤔 

Date is still less confusing but is it created, update, start or end? I think we just need to add some more context to these options so it is clear to users what they are sorting on and have an understanding of the expected outcome.

Reply:
Spun out issue #6471